.ci/deps.sh

```diff
@@ -67,8 +67,8 @@ fi
 
 deps_perl="perl libperl-critic-perl"
 
-sudo apt-get -y update
-sudo apt-get -y --no-install-recommends install $deps $deps_perl $deps_infer
+sudo apt-get -y update|grep -v '\%'
```

Comment:
why don't you just redirect it to /dev/null? imo it's clearer what you're trying to do than this.